libtcmu_log.c

```diff
@@ -107,6 +107,7 @@ void tcmu_set_log_level(int level)
 	else if (level < TCMU_CONF_LOG_LEVEL_MIN)
 		level = TCMU_CONF_LOG_LEVEL_MIN;
 
+	tcmu_info("log level now is %d\n", level);
```

Comment:
Lets covert the 'level' to string form and then print. That way it makes more sense.

Reply:
Yeah, make sense.